### ISSUE TITLE
Core Web Vitals Assessment Failed

### ISSUE THREAD

**Issue by ronnievsmith** [material-components/material-components-site-generator#46]
Just an FYI.

Just exploring Page Speed Insights for Material 3 Web and thought I would test the overall Material documentation page...

<img width="744" alt="image" src="https://github.com/material-components/material-components-site-generator/assets/12751611/053c962e-04c3-4537-89bb-7d723ea064e7">

**Comment by ronnievsmith** [comment]
Still an F?

<img width="614" alt="image" src="https://github.com/material-components/material-components-site-generator/assets/12751611/d68ea540-d224-4a6f-a36e-c63bf91f13e5">

**Comment by Maxim-Mazurok** [comment]
I don't think this builder is used for m3.material.io... Doesn't look like it's used for m2.material.io either... Was comparing `jekyll-site-src/_includes/head.html` and the HTML in production
